### ISSUE TITLE
wp.pl

### ISSUE THREAD

**Issue by xxcriticxx** [X01X012013/AdBlockProtector#38]
ad block on video can you please check 

`http://www.pudelek.tv/video/Denerwuje-mnie-jak-gwiazdy-UDAJA-ZE-SA-CHARYTATYWNE-Pokazuja-kreacje-na-czerwonym-dywanie-A-POZNIEJ-ZERO-14206/`

and

`http://www.money.pl/gospodarka/wiadomosci/artykul/drony-dla-wojska-zakup-dronow-dla-wojska,45,0,2211629.html`

**Comment by mapx-** [comment]
add
```
@@||pagead2.googlesyndication.com^$domain=pudelek.tv
@@||pagead2.googlesyndication.com^$domain=money.pl
```

**Comment by X01X012013** [comment]
Hi, this is fixed with rules 3.1. Thank you for your contribution.

**Comment by X01X012013** [comment]
Since there is no follow up, I assume this issue is fixed.
